pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -52,6 +52,11 @@ func init() {
 }
 
 func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
+	hostOnlyCIDR = strings.Trim(hostOnlyCIDR, "'\"")
```

Comment:
I think we only want to trim single quotes

Reply:
Thanks. I'll change it